tests/testthat/test-breaks-log.r

```diff
@@ -62,3 +62,16 @@ test_that("log_breaks arguments are forcely evaluated on each call #81", {
   expect_equal(subfun1(1:1000), subfuns[[1]](1:1000))
   expect_equal(subfun2(1:1000), subfuns[[2]](1:1000))
 })
+
+test_that("log_breaks with very small ranges fall back to extended_breaks", {
+  expect_equal(log_breaks(n = 5, base = 10)(c(2.001, 2.002)),
```

Comment:
Oops, I missed that these don't use the tidyverse indent style: http://style.tidyverse.org/syntax.html#long-lines

Reply:
I succumbed and installed the `styler` package.  :)